sockets/marketWatcher.js

```diff
@@ -1,13 +1,16 @@
-
-
 const api = require('../lib/api');
-const logger = require('../utils/logger');
+// const logger = require('../utils/logger');
```

Comment:
why are we disabling logger here?

Reply:
@Isabello TBMU, we don't use it in this file. it's imported, used in a function definition but not used/called anywhere.